packages/web/src/components/track/desktop/TrackTile.tsx

```diff
@@ -222,38 +282,32 @@ const TrackTile = ({
       >
         {artwork}
       </div>
-      {showPremiumDogTag && dogEarType ? (
-        <DogEar
-          type={dogEarType}
-          containerClassName={styles.premiumDogEarContainer}
-        />
-      ) : null}
-      {isArtistPick && !showPremiumDogTag ? (
-        <DogEar type={DogEarType.STAR} />
-      ) : null}
-      {isUnlisted && <DogEar type={DogEarType.HIDDEN} />}
+      {dogEarType && <DogEar type={dogEarType} />}
```

Comment:
I think in our style guide we're trying to move away from `&&` in favor of ternaries.
https://www.notion.so/audiusproject/Javascript-Typescript-Code-Style-4f34640139fc4a809c3765131bc597ff?pvs=4#f1c89a4e0843430bbf98f0ce53dbcf14

Reply:
oh jeez, I thought the style guide went the other way. Will change